lms/djangoapps/survey/utils.py

```diff
@@ -1,41 +1,44 @@
 """
-Helper methods for Surveys
+Utilities for determining whether or not a survey needs to be completed.
 """
-
 from courseware.access import has_access
-from survey.models import SurveyAnswer, SurveyForm
+from survey.models import SurveyForm, SurveyAnswer
 
 
 def is_survey_required_for_course(course_descriptor):
     """
     Returns whether a Survey is required for this course
     """
 
-    # check to see that the Survey name has been defined in the CourseDescriptor
-    # and that the specified Survey exists
+    # check to see that the survey is required in the CourseDescriptor
+    is_survey_required = getattr(course_descriptor, 'course_survey_required', False)
 
-    return course_descriptor.course_survey_required and \
-        SurveyForm.get(course_descriptor.course_survey_name, throw_if_not_found=False)
+    #  that the specified Survey exists
+    is_survey_required = (is_survey_required and
```

Comment:
I'd either use the code I previously wrote, or use `&=`.  Not sure if everyone likes it, but I find it simpler to read.
```
is_survey_required &= SurveyForm.get(course_descriptor.course_survey_name, throw_if_not_found=False)
```